### PR TITLE
Solved issue #3533 by editing the label "create a new Twitter applica…

### PR DESCRIPTION
**Pull request by Jenniline**
…tion"

Solved issue #3533 by editing the label "create a  new Twitter application"

I added the sentences "Twitter applications may take a few hours or a few days to be approved by Twitter. Please be aware of this if you need this data quickly.',.

This pull request makes the following changes:
-Adds a label that informs users of the lengthy process involved in create a Twitter application

Test checklist:
- [ ]

- [ ] I certify that I ran my checklist

Fixes ushahidi/platform# .

Ping @ushahidi/platform

### CODE REVIEW

File: src/App/DataSource/Twitter/Twitter.php [hunk 1]
```diff
@@ -71,9 +71,10 @@ public function getOptions()
     {
         return [
             'intro_step1' => [
-                'label' => 'Step 1: Create a new Twitter application',
+                'label' => 'Step 1: Create a new Twitter application.<br /> "Twitter applications may take a few hours or a few days to be approved by Twitter. <br />Please be aware of this if you need this data quickly.',
```
Comment (Angamanga) [inline src/App/DataSource/Twitter/Twitter.php]: @Jenniline Thank you for making this change! Could you just remove the " before Twitter? Thank you 🎉

Comment (Jenniline) [reply src/App/DataSource/Twitter/Twitter.php]: Alright. I am on it

**Comment by Angamanga** [comment]
@Jenniline One last thing, can you change the text to "few hours or days"? Thank you!

**Comment by Angamanga** [comment]
@Jenniline Also, could you remove the <br><br />? They show up as text in the view.

**Comment by Jenniline** [comment]
Hi Anna, I don't see any "?" do you mean the "<br />" ?

**Comment by Jenniline** [comment]
Hi, Anna I just made the Pull request please check it out. Thanks

**Comment by Angamanga** [comment]
@Jenniline I meant `<br>` and `<br/>`. I don't know why it showed up as ? :)

**Comment by Jenniline** [comment]
Hi Anna, I had imagined that is what you meant and I made the change yesterday. The pull request is still here under this same branch. 
Thanks

**Comment by tuxpiper** [comment]
@all-contributors please add @Jenniline for code

**Pull request merged**